src/lib/components/Select.react.js

```diff
@@ -0,0 +1,123 @@
+// @flow
+
+import React, {Component} from 'react';
+import { default as MuiSelect } from '@material-ui/core/Select';
```

Comment:
The package actually does have the Select component, but we call it the dropdown menu. I'll see if I can help you get this running properly to build everything so that you'll be ready if you want to contribute any other components.

Reply:
Oh, I think the dropdown menu was this [menu](https://material-ui.com/components/menus/). But yeah, I'll try to run this properly first, so I can check the other options to contribute.